executor/distsql.go

```diff
@@ -126,6 +126,8 @@ func zone(sctx sessionctx.Context) (string, int64) {
 	_, offset := time.Now().In(loc).Zone()
 	var name string
 	name = loc.String()
+	// when we found name is "Local", we have no chice but push down
```

Comment:
why?

Reply:
`scCtx.GetSessionVars().Location()` calls `timeutil.Local()` if timezone of session is nil. `timeutil.Local()` tried to retrieve IANA timezone name from `TZ` and `/etc/localtime` already. 
If it still returns a `time.Local`, then we have no choice but to push down `time.Local` to `tikv`.